### PR TITLE
Revert rules when the process is killed. Added quiet, less quiet modes

### PR DESCRIPTION
**Pull request by kxalex**
* There was the issue if `holepunch` is used with command and the python process is killed the rules are not reverted.
   Added termination handler for SIGINT, SIGTERM, and SIGHUP which properly exists (i.e. revert rules).

* Added quiet and less verbose modes.

### CODE REVIEW

File: holepunch/__init__.py [hunk 1]
```diff
@@ -315,7 +329,15 @@ def holepunch(args):
 
     command = args['--command']
     if command is not None:
-        print('Rules will revert when `%s` terminates.' % command)
+        log('Rules will revert when `%s` terminates.' % command)
+
+        # noinspection PyUnusedLocal
+        def terminate(num, frame):
```
Comment (erik) [inline holepunch/__init__.py:335]: let's move this next to the `atexit.register` call so all the cleanup logic is in one place.

File: README.md [hunk 2]
```diff
@@ -62,3 +62,13 @@ holepunch foo_bar --cidr=192.168.0.0/16 22 80
...
+
+Less verbose or quiet mode:
+
+```
+# Verbose mode
+holepunch foo_bar -v2 --cidr=192.168.0.0/16 22 80
+
+# Quiet mode
+holepunch foo_bar -v0 --cidr=2001:882f::1/128 443
+```
```
Comment (erik) [inline README.md:74]: Overall I like this idea, but let's split it out into its own PR so we can address it separately from the bug fix.

**Review by erik — CHANGES_REQUESTED** [review]
Great catch - my assumption when writing that initially was that the `atexit.register` call would handle interrupts, but clearly that isn't true.

Would appreciate if you split out the unrelated change for verbose/quiet mode to a separate PR before we land this

File: README.md (hunk 2 above)
Comment (kxalex) [reply README.md:74]: Sure. I'll create another PR for verbose mode

Comment (kxalex) [reply README.md:74]: @erik created PR for verbose mode. Let me know what else I need to update. PR https://github.com/erik/holepunch/pull/19

**Comment by kxalex** [comment]
Split into #19 and #20
